features/admin.users.v1/pages/users.tsx

```diff
@@ -760,7 +764,16 @@ const UsersPage: FunctionComponent<UsersPageInterface> = (
             dropDownOptions.push({
                 "data-componentid": `${componentId}-invite-parent-user`,
                 key: 3,
-                text: t("parentOrgInvitations:addUserWizard.heading"),
+                text: <>
+                    { t("parentOrgInvitations:addUserWizard.heading") }
+                    { showStatusLabelForNewAuthzRuntimeFeatures && !legacyAuthzRuntime && (
+                        <Chip
+                            size="small"
+                            label={ t(FeatureStatusLabel.NEW).toUpperCase() }
+                            className="oxygen-chip-new mb-1 ml-2"
```

Comment:
Let's avoid mb-1 and ml-2 styles wherever possible

Reply:
Fixed with [3c6d4f2](https://github.com/wso2/identity-apps/pull/6256/commits/3c6d4f2fcf089e1a2eaa7627241d72069850fcc4)